{"uuid":"0cdca150-84a7-417d-a527-9d450d9fa562","name":"com.mobile.app.Base.BaseTest.setUp","children":["658e2c59-3f5a-48b3-8cef-da95537bee36"],"description":"","befores":[{"name":"setUp","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The local appium server has not been started. The given Node.js executable: C:\\Program Files\\nodejs\\node.exe Arguments: [C:\\Program Files\\nodejs\\node_modules\\appium\\build\\lib\\main.js, --port, 4724, --address, 127.0.0.1] \nProcess output: \u001B[38;5;112m[Appium]\u001B[0m Welcome to Appium v2.12.1\r\n\u001B[38;5;112m[Appium]\u001B[0m Non-default server args:\r\n\u001B[38;5;112m[Appium]\u001B[0m { address: \u001B[32m'127.0.0.1'\u001B[39m, port: \u001B[33m4724\u001B[39m }\r\n\u001B[38;5;112m[Appium]\u001B[0m The autodetected Appium home path: C:\\Users\\crroj\\.appium\r\n\u001B[38;5;112m[Appium]\u001B[0m Attempting to load driver uiautomator2...\r\n\u001B[38;5;112m[Appium]\u001B[0m Requiring driver at C:\\Users\\crroj\\.appium\\node_modules\\appium-uiautomator2-driver\\build\\index.js\r\n\u001B[38;5;112m[Appium]\u001B[0m AndroidUiautomator2Driver has been successfully loaded in 0.913s\r\n\u001B[38;5;86m[HTTP]\u001B[0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.\r\n\u001B[38;5;112m[Appium]\u001B[0m Could not configure Appium server. It's possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4724\r\n\u001B[38;5;112m[Appium]\u001B[0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4724\n    at Server.setupListenHandle [as _listen2] (node:net:1907:16)\n    at listenInCluster (node:net:1964:12)\n    at node:net:2170:7\n    at processTicksAndRejections (node:internal/process/task_queues:90:21)\r\n\n","trace":"io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: The local appium server has not been started. The given Node.js executable: C:\\Program Files\\nodejs\\node.exe Arguments: [C:\\Program Files\\nodejs\\node_modules\\appium\\build\\lib\\main.js, --port, 4724, --address, 127.0.0.1] \nProcess output: \u001B[38;5;112m[Appium]\u001B[0m Welcome to Appium v2.12.1\r\n\u001B[38;5;112m[Appium]\u001B[0m Non-default server args:\r\n\u001B[38;5;112m[Appium]\u001B[0m { address: \u001B[32m'127.0.0.1'\u001B[39m, port: \u001B[33m4724\u001B[39m }\r\n\u001B[38;5;112m[Appium]\u001B[0m The autodetected Appium home path: C:\\Users\\crroj\\.appium\r\n\u001B[38;5;112m[Appium]\u001B[0m Attempting to load driver uiautomator2...\r\n\u001B[38;5;112m[Appium]\u001B[0m Requiring driver at C:\\Users\\crroj\\.appium\\node_modules\\appium-uiautomator2-driver\\build\\index.js\r\n\u001B[38;5;112m[Appium]\u001B[0m AndroidUiautomator2Driver has been successfully loaded in 0.913s\r\n\u001B[38;5;86m[HTTP]\u001B[0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.\r\n\u001B[38;5;112m[Appium]\u001B[0m Could not configure Appium server. It's possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4724\r\n\u001B[38;5;112m[Appium]\u001B[0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4724\n    at Server.setupListenHandle [as _listen2] (node:net:1907:16)\n    at listenInCluster (node:net:1964:12)\n    at node:net:2170:7\n    at processTicksAndRejections (node:internal/process/task_queues:90:21)\r\n\n\r\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:190)\r\n\tat com.mobile.app.Base.BaseTest.startAppiumService(BaseTest.java:38)\r\n\tat com.mobile.app.Base.BaseTest.setUp(BaseTest.java:24)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\r\n\tat org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:833)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:600)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)\r\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\r\n\tat java.base/java.lang.Thread.run(Thread.java:833)\r\nCaused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://127.0.0.1:4724/status] to be available after 20010 ms\r\n\tat org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:89)\r\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:155)\r\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:177)\r\n\t... 28 more\r\nCaused by: java.util.concurrent.TimeoutException\r\n\tat java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)\r\n\tat org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:87)\r\n\t... 30 more\r\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1749523357055,"stop":1749523377430}],"afters":[],"start":1749523357054,"stop":1749523377431}